module/Company/src/Company/Form/EditCompany.php

```diff
@@ -0,0 +1,317 @@
+            'type' => 'Zend\Form\Element\Url',
+            'attributes' => array(
+            ),
+            'options' => array(
+                'label' => $translate->translate('Website'),
+            ),
+        ));
+        $this->add(array(
+            'name' => 'en_slogan',
+            'attributes' => array(
+                'type' => 'text',
+            ),
+            'options' => array(
+                'label' => $translate->translate('Slogan'),
+            ),
+        ));
+        $this->add(array(
+            'name' => 'nl_slogan',
+            'attributes' => array(
+                'type' => 'text',
+            ),
+            'options' => array(
+                'label' => $translate->translate('Slogan'),
+            ),
+        ));
+        $this->add(array(
+            'name' => 'email',
+            'type' => 'Zend\Form\Element\Email',
+            'attributes' => array(
+                'required' => 'required',
+            ),
+            'options' => array(
+                'label' => $translate->translate('Email'),
+                'required' => 'required',
+            ),
+        ));
+        $this->add(array(
+            'name' => 'phone',
+            'attributes' => array(
+                'type' => 'text',
+            ),
+            'options' => array(
+                'label' => $translate->translate('Phone'),
+            ),
+        ));
+        $this->add(array(
+            'name' => 'nl_logo',
+            'attributes' => array(
+                'type' => 'file',
+            ),
+            'options' => array(
+                'label' => $translate->translate('Logo'),
+            ),
+        ));
+        $this->add(array(
+            'name' => 'en_logo',
+            'attributes' => array(
+                'type' => 'file',
+            ),
+            'options' => array(
+                'label' => $translate->translate('Logo'),
+            ),
+        ));
+        $this->add(array(
+            'name' => 'en_description',
+            'type' => 'Zend\Form\Element\Textarea',
+            'options' => array(
+                'label' => $translate->translate('Description'),
+            ),
+            'attributes' => array(
+                'type' => 'textarea',
+            ),
+        ));
+        $this->add(array(
+            'name' => 'nl_description',
+            'type' => 'Zend\Form\Element\Textarea',
+            'options' => array(
+                'label' => $translate->translate('Description'),
+            ),
+            'attributes' => array(
+                'type' => 'textarea',
+            ),
+        ));
+
+        $this->add(array(
+            'name' => 'submit',
+            'attributes' => array(
+                'type' => 'submit',
+                'value' => $translate->translate('Submit changes'),
+                'id' => 'submitbutton',
+            ),
+        ));
+
+        $this->initFilters();
+    }
+
+    protected function initFilters()
```

Comment:
It is better to use a `InputFilterProviderInterface` and give an input filter specification here.